### PR TITLE
CPP-195 - Finished support for libuv v1.x

### PR DESCRIPTION
**Pull request by mpenick**

### CODE REVIEW

File: src/address.hpp [hunk 1]
```diff
@@ -42,6 +42,10 @@ class Address {
 
   bool init(const struct sockaddr* addr);
 
+  const struct sockaddr* addr() const {
+    return copy_cast<const struct sockaddr_storage*, const struct sockaddr*>(&addr_);
+  }
+
```
Comment (aholmberg) [inline src/address.hpp]: Curious why this was added.

Comment (aholmberg) [inline src/address.hpp]: Never mind. IDE search fail was not showing usage.

File: src/loop_thread.hpp [hunk 2]
```diff
@@ -26,7 +26,6 @@ class LoopThread {
 public:
   LoopThread()
       : loop_(uv_loop_new())
```
Comment (aholmberg) [inline src/loop_thread.hpp]: It  looks like we're still using the deprecated loop functions here. Did you intend to change that for libuv 1.x to address [CPP-194](https://datastax-oss.atlassian.net/browse/CPP-194)?

Comment (mpenick) [inline src/loop_thread.hpp]: That's fixed in this PR: https://github.com/datastax/cpp-driver/pull/126

Comment (aholmberg) [inline src/loop_thread.hpp]: Ok. You never mentioned that PR in CPP-194, so I thought this was the only one.

**Pull request merged**